content/blog/2018/07/2018-07-12-simple-pull-request-plugin.adoc

```diff
@@ -0,0 +1,193 @@
+---
+layout: post
+title: "Pipeline as YAML: Alpha release"
+tags:
+- gsoc2018
+- plugin
+- pipeline
+- yaml
+author: abhishek_gautam
+---
+
+=== About me
+
+I am Abhishek Gautam, 3rd year student from Visvesvaraya National Institute of
+technology, India, Nagpur. I was a member of ACM Chapter and Google student developer club of my
+college. I am passionate about automation.
+
+=== Mentors
+
+* link:https://github.com/oleg-nenashev[Oleg Nenashev] (Org Admin)
+* link:https://github.com/martinda[Martin d'Anjou]
+* link:https://github.com/kwhetstone[Kristin Whetstone]
+* link:https://github.com/grandvizier[Jeff Knurek]
+
+=== Project Summary
+
+This is a GSoC 2018 project.
```

Comment:
Maybe it makes sense to remove the mentors section above and add a "My mentors are: Ab, B, C, D" here. Just to shorten the intro